R/Spectra.R

```diff
@@ -1028,6 +1028,8 @@ setMethod("filterPrecursorScan", "Spectra",
 #' @rdname Spectra
 setMethod("filterRt", "Spectra",
           function(object, rt = numeric(), msLevel. = unique(msLevel(object))) {
+              if (!identical(length(rt), 2L) || rt[1] >= rt[2])
+                  stop("Please provide a lower and upper retention time range.")
               suppressWarnings(rt <- range(rt))
```

Comment:
Just a comment: I used here the `range(rt)` to silently ensure that length rt is always 2 and that `rt[2] > rt[1]`. If you prefer to have the explicit check here we can remove that line.

Reply:
This PR came up because I initially tried it with a single value, and I hadn't been warned of my mistake, which would be beneficial, I think.

Reply:
I'm OK with that.

Reply:
If we ensure that `rt` is numeric and in the correct order by the `if` statement above do we really need `suppressWarning` here. Also `range` should be superfluous. IMHO we can remove the whole line.
```suggestion
```